apps/explorer/src/pages/transaction-result/programmable-transaction-view/Transaction.tsx

```diff
@@ -60,23 +67,37 @@ function MoveCall({ type, data }: TransactionProps<MoveCallSuiTransaction>) {
         arguments: args,
         type_arguments: typeArgs,
     } = data;
+
+    const stringData = `(package: ${movePackage}, module: ${module}, function: ${func}, arguments: ${args}, type_arguments: ${typeArgs})`;
+
     return (
-        <TransactionContent type={type}>
-            (package: <ObjectLink objectId={movePackage} />, module:{' '}
-            <ObjectLink
-                objectId={`${movePackage}?module=${module}`}
-                label={`'${module}'`}
-            />
-            , function: <span className="break-all text-sui-dark">{func}</span>
-            {args && (
-                <span className="break-all">
-                    , arguments: [{flattenSuiArguments(args!)}]
-                </span>
-            )}
-            {typeArgs && (
-                <span className="break-all">, type_arguments: {typeArgs}</span>
-            )}
-            )
+        <TransactionContent>
+            <div className="flex justify-between gap-7">
+                <div>
+                    (package: <ObjectLink objectId={movePackage} />, module:{' '}
+                    <ObjectLink
+                        objectId={`${movePackage}?module=${module}`}
+                        label={`'${module}'`}
```

Comment:
Why do we keep quote-wrapping the module? this has never made sense to me.

Reply:
@mystie711 ^ could you comment

Reply:
Before I go and rename it everywhere in this PR, what should we call this one. I have consulted with ChatGPT and it tells me to use `MistyEdge`, can we go with this @mystie711 ?

<img width="714" alt="Screenshot 2023-04-28 at 10 27 37 PM" src="https://user-images.githubusercontent.com/127577476/235285360-3f78f3af-46e0-4185-8efe-ed4d59349af0.png">

CC: @Jordan-Mysten